src/material/chips/chips.md

```diff
@@ -187,3 +187,5 @@ For both MatChipGrid and MatChipListbox, always apply an accessible label to the
 Always apply MatChipRemove to a `<button>` element, never a `<mat-icon>` element.
 
 When using MatChipListbox, never nest other interactive controls inside of the `<mat-chip-option>` element. Nesting controls degrades the experience for assistive technology users.
+
+By default, `MatChipListbox` displays a checkmark to identify selected items. While you can the checkmark indicator for single-selection via `hideSingleSelectionIndicator`, this makes your component less accessible by making it harder or impossible for users to visually identify selected items.
```

Comment:
"can the checkmark" => "can disable the checkmark"

"this makes your component" => "this makes the component"

Reply:
fixed the typos